VespaGAS/ViewModels/GasCalculatorViewModel.swift

```diff
@@ -1,5 +1,36 @@
 import Foundation
+import RxSwift
 
-public struct GasCalculatorViewModel {
+struct GasCalculatorViewModel {
+    private let disposeBag = DisposeBag()
+    private let oilValueSubject = BehaviorSubject(value: "0%")
```

Comment:
Considering you're using it to map values from the oilPercentageObservable, wouldn't a simple map of the input observable with a `startWith` do the job?

So something like:

``` swift
self.oilValueObservable = oilPercentageObservable.map { "\(Int($0))%"}.startWith("0%")
```